lib/chromic_pdf/pdf/connection/inet.ex

```diff
@@ -0,0 +1,75 @@
+# SPDX-License-Identifier: Apache-2.0
+
+if Code.ensure_loaded?(WebSockex) do
+  defmodule ChromicPDF.Connection.Inet do
```

Comment:
Just a 💅 : if the application is called `inets` maybe the module should also be called `Inets`?

Reply:
The erlang application is called `:inets` (contains `:httpc` among other utilities). This module is called `Inet` to mean "goes through the network". With `inet` being the usual term for anything IP-based?

Reply:
`ChromicPDF.Connection.Remote` would be a good counterpart for `ChromicPDF.Connection.Local`.
Or can there be other "remote" protocols than this?

Reply:
:thinking: true... Problem with `remote` is that it's already a bit overloaded, as the entire mechanism is called "remote debugging" in Chrome (see the `--remote-debugging-pipe` switch), which just means that the inspector is controlled from "outside the Chrome process"... So I was also reluctant to call the section "Remote Chrome" in the docs

Reply:
Fair enough, I didn't realize it.